suggested Tags added to to photic stimulation

tpatpa opened the pull request:
Added suggestedTags to Intermittent-photic-stimulation according to Table 7. Effect of the intermittent photic stimulation.

Schema had no validation issues and was successfully converted using HED tools.

VisLab reviewed (commented):
This is okay, but I would suggest using the parent node Intermittent-photic-stimulation-effect rather than all of the individual nodes -- this is the strategy that is used in the standard schema.
Tools will show all of the items in the hierarchy.

tpatpa commented:
Thank you @VisLab !
Changed accordingly.

tpatpa commented:
Thank you @VisLab !

I have replaced relevant occurrences with the following parent nodes:
 - `suggestedTag=Rhythmic-activity-morphology`
 - `suggestedTag=Periodic-discharges-morphology`
 - `suggestedTag=Posterior-dominant-rhythm-property`
 - `suggestedTag=Periodic-discharge-time-related-features`

I have also changed all occurrences of `Periodic-discharges-` to `Periodic-discharge-`.
